plugins/decklink/decklink-output.cpp

```diff
@@ -208,6 +208,8 @@ static obs_properties_t *decklink_output_properties(void *unused)
 	obs_properties_add_list(props,
 			MODE_ID, TEXT_MODE, OBS_COMBO_TYPE_LIST, OBS_COMBO_FORMAT_INT);
 
+	obs_properties_add_bool(props, AUTO_START, TEXT_AUTO_START);
```

Comment:
Maybe move this line up to group it with the other `obs_properties_add_*` call?  Up to you.